src/Tribe/RSVP.php

```diff
@@ -2037,7 +2074,7 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 			}
 		}
 
-		$attendee_data = array_merge( $this->get_order_data( $attendee->ID ), array(
+		$attendee_data = array_merge( $this->get_order_data( $attendee->ID ), [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: d971d5eda509428acf64110050647961</sub>